youtube_dl/postprocessor/embedthumbnail.py

```diff
@@ -56,38 +57,25 @@ def run(self, info):
             os.rename(encodeFilename(temp_filename), encodeFilename(filename))
 
         elif info['ext'] in ['m4a', 'mp4']:
-            if not check_executable('AtomicParsley', ['-v']):
```

Comment:
Do not touch existing `AtomicParsley` code. `mutagen` path must be a fallback if `AtomicParsley` is missing.

Reply:
Ok, this should be good too.